securedrop-log/debian/securedrop-log.install

```diff
@@ -0,0 +1 @@
+securedrop.Log etc/qubes-rpc/securedrop.Log
```

Comment:
I think this will create a _folder_, i.e. `/etc/qubes-rpc/securedrop.Log/securedrop.Log`, instead what about:

```suggestion
securedrop.Log etc/qubes-rpc/
```

Reply:
fixing this.